.travis.yml

```diff
@@ -2,14 +2,12 @@ language: java
 
 sudo: false
 
-script:
-  - jdk_switcher use oraclejdk7
-  - mvn clean verify -P jdk7
-  - jdk_switcher use oraclejdk8
-  - mvn clean verify -P jdk8
+jdk:
```

Comment:
This makes travis run the tests for both JDKs in parallel, which is already a big improvement.